test/Kozani/KozaniManagerRuntimeTests/KozaniManagerRuntimeTests.cpp

```diff
@@ -2,12 +2,70 @@
 // Licensed under the MIT License.
 
 #include "pch.h"
+#include <KozaniManager_h.h>
+#include "winrt/Microsoft.Kozani.ManagerRuntime.h"
 
 namespace TB = ::Test::Bootstrap;
 namespace TP = ::Test::Packages;
 
+namespace KozaniManagerTestPackage
+{
+    constexpr PCWSTR c_PackageDirName = L"KozaniManager";
+    constexpr PCWSTR c_PackageFullName = L"KozaniManager_1.0.0.0_neutral__8wekyb3d8bbwe";
+}
+
 namespace Test::KozaniManagerRuntimeTests
 {
+    struct MyKozaniStatusCallback : winrt::implements<MyKozaniStatusCallback, IKozaniStausCallback, winrt::Windows::Foundation::IInspectable>
+    {
+#pragma region IKozaniStausCallback_methods
+        HRESULT OnActivated(DWORD pid)
+        {
+            WEX::Logging::Log::Comment(WEX::Common::String().Format(L"IKozaniStausCallback::OnActivated is called. pid = %u", pid));
+            m_isActivated = true;
+            return S_OK;
+        }
+
+        HRESULT OnActivationFailed(
+            DWORD errorCode,
+            PCWSTR errorMessage)
+        {
+            WEX::Logging::Log::Comment(
+                WEX::Common::String().Format(L"IKozaniStausCallback::OnActivationFailed is called. errorCode = 0x%x, errorMessage: %s",
+                    errorCode, errorMessage));
+            m_isActivationFailed = true;
+            return S_OK;
+        }
+
+        HRESULT OnClosed()
+        {
+            WEX::Logging::Log::Comment(L"IKozaniStausCallback::OnClosed is called.");
+            m_isClosed = true;
+            return S_OK;
+        }
+#pragma endregion IKozaniStausCallback_methods
+
+        bool IsActivated()
+        {
+            return m_isActivated;
+        }
+
+        bool IsActivationFailed()
+        {
+            return m_isActivationFailed;
+        }
+
+        bool IsClosed()
+        {
+            return m_isClosed;
+        }
+
+    private:
+        bool m_isActivated;
```

Comment:
NIT: `bool m_isActivated{};`

Ensure locals and attributes are initialized and ES.23: [Prefer the {}-initializer syntax](https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines.html#es23-prefer-the--initializer-syntax)

same comment applies throughout

Reply:
Member variables should be by default set to default value. Anyway, added {}.